10-JavaScript/testing.md

```diff
@@ -23,96 +22,119 @@ In short, most interesting JavaScript code is full of *external dependencies*. M
 
 ```javascript
-describe('Word', () => {
-  describe('constructor', () => {
+describe('Pangram class',  () =>  {
+
+
+  describe('isPangram()', () => {
 
   });
 });
 ```
 
-#### `it` Blocks
+#### `test` Blocks
 
-`it` blocks should define one distinct test. The description that goes along with the `it` block should describe the specific scenario that you are testing.
+`test` blocks should define one distinct test. The description that goes along with the `test` block should describe the specific scenario that you are testing.
 
 ```javascript
-describe('Word', () => {
-  describe('constructor', () => {
-    it ('Converts text to lowercase', () => {
+describe('Pangram class',  () =>  {
+  test('Pangram is defined', function() {
+
+  }
+
+  describe('isPangram()', () => {
 
-    });
   });
 });
 ```
 
+Similar to what you have in Minitest you can use `it`, in place of `test` if `it` (hic) makes you feel more comfortable, but the Jest documentation uses `test`, and so Ada's examples will as well.
+
 #### `expect` Statements
 
 Expectations should be the "meat and potatoes" of your tests, inside of your `it` blocks. Each test has at least one `expect` statement to ensure the behavior is as expected.
 
-The syntax of `describe` and `it` is pretty similar to Minitest (at least as similar as Ruby and JS can be), but `expect` is somewhat different. Note that there are actually two functions being called, the `expect` and the matcher (`toEqual` in this case), each taking one argument.
+The syntax of `describe` and `test` is pretty similar to Minitest (at least as similar as Ruby and JS can be), but `expect` is somewhat different. Note that there are actually two functions being called, the `expect` and the matcher (`toBeDefined` in this case).
 
 ```javascript
-describe('Word', () => {
-  describe('constructor', () => {
-    it ('Converts text to lowercase', () => {
-      const word = new Word({ text: 'TeSt' });
-      expect(word.get('text')).toEqual('test');
-    });
+describe('Pangram class',  () =>  {
+  test('Pangram is defined', function() {
+    expect(Pangram).toBeDefined();
+  });
+
+  describe('isPangram()', () => {
+
   });
 });
 ```
 
 And that's all a test is. Go ahead and add another one to the spec file, but this time make sure it will fail (e.g. `expect(false).toEqual(true);`). Then re-run the tests, just to see what a failure looks like. Fix the test and run it again. Not too different from Minitest, right?
 
+## Skipping Tests
+
+Tests can be skipped by changing `test(...` to `test.skip(...`.  Go ahead and change
+
+```javascript
+    test('isPangram() is defined', () => {
+```
+
+to:
+
+```javascript
+test.skip('isPangram() is defined', () => {
+```
+
+If you choose to use `it` blocks instead of `test`, you can skip a test with `xit` in place of `it`.
```

Comment:
I think we should just leave this out with the assumption that we're going to focus on using `test` over `it`